### PR TITLE
Rds/kjw/rule 21 14

### PR DESCRIPTION
**Pull request by KarenWGard**

### CODE REVIEW

File: docs/section21/Rule_21-14.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 21 Boiler  
+**Appendix G Section Reference:** Section G3.1.1.1 & G3.1.1.3.1 Building System-Specific Modeling Requirements for the Baseline model  
+**Schema Version:** 0.0.25
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. P-RMR is modeled with purchased hot water or steam.
+
```
Comment (supriyagoel) [inline docs/section21/Rule_21-14.md:15]: need to replace RMR with RMI

File: docs/section21/Rule_21-14.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:** 
+
+1. check_purchased_chw_hhw()
+
+**Applicability Checks:**
+
+- Check if B-RMR is modeled with purchased chilled water or purchased hot water/steam: `purchased_chw_hhw_status_dict = check_purchased_chw_hhw(B_RMR)`
+
```
Comment (supriyagoel) [inline docs/section21/Rule_21-14.md:26]: this doesnt need to check purchased chilled water, correct?

File: docs/section21/Rule_21-14.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  - If B-RMR is not modeled with purchased hot water/steam, rule is not applicable to B-RMR: `if NOT purchased_chw_hhw_status_dict["PURCHASED_HEATING"]: RULE_NOT_APPLICABLE`
+
+  - Else, B-RMR is modeled with purchased hot water/steam, continue to rule logic: `else: CHECK_RULE_LOGIC`
+
+## Rule Logic:  
+
+**Rule Assertion:**
+
+- Case 1: `UNDETERMINED and raise_message "P-RMR IS MODELED WITH PURCHASED HOT WATER OR STEAM. VERIFY B-RMR HOT WATER RESET IS NOT MODELED."`
+
```
Comment (supriyagoel) [inline docs/section21/Rule_21-14.md:36]: can update the message to say 'proposed building is modeled with purchased hot water or steam. Verify that hot water reset is not modeled in baseline model.'

Comment (KarenWGard) [reply docs/section21/Rule_21-14.md:36]: updated

File: docs/section21/Rule_21-14.md (hunk 2 above)
Comment (KarenWGard) [reply docs/section21/Rule_21-14.md:26]: right.  updated

File: docs/section21/Rule_21-14.md (hunk 1 above)
Comment (KarenWGard) [reply docs/section21/Rule_21-14.md:15]: updated

**Review by supriyagoel — APPROVED** [review]

**Review by weilixu — APPROVED** [review]

**Pull request merged**